post.md

```diff
@@ -27,17 +27,17 @@ I remember feeling slightly patronised by that.
 Why can't I, as an _application author_, become master of my own destiny and make use of this powerful feature?
 Is it too complex for mere mortals to comprehend?
```

Comment:
Yes. haha. There's a lot of this inner voice and it's quite funny (funny haha) but also a little angry. Were you angry when you wrote this??? You become a data fetching framework author to spite them???